CHANGES_NEXT_RELEASE

```diff
@@ -4,3 +4,25 @@
 - Fix: fix issue with unrecognized NGSILD-Tenant and NGSILD-Path headers
 - Fix: mongodb dependency moved to devDependencies (as it is used only in test code)
 - Upgrade mongodb dep from 3.6.12 to 4.7.0
+- Upgrade body-parser dependency from 1.19.0 to 1.20.0
+- Upgrade express dependency from 4.16.4 to 4.18.1
+- Upgrade got dependency from 11.8.2 to 11.8.3
```

Comment:
```suggestion
- Upgrade got dependency from 11.8.2 to 11.8.5
```

Reply:
I think that one covered two of the vulnerabilities.